style.css

```diff
@@ -1,45 +1,111 @@
-content{
+
+body{
   margin: 0;
+  font-family: sans-serif;
+  font-size: 1em;
+}
+
+* {
+  box-sizing: borderbox;
 }
+
 .navbar {
-  display: block;
-  background: teal;
-  overflow:auto;
+  background: #109094;
   color: white;
+  overflow:auto;
 }
 
 .navbar_title {
   float: left;
-  padding: 2rem;
+  padding-bottom: .5rem;
+  padding-left: .5rem;
 }
+
 .navbar_people {
   float: right;
-  padding: 2rem;
+  padding-bottom: .5rem;
+  padding-right: .5rem;
+  padding-left: .5rem;
 }
 
 .navbar_repos {
   float: right;
-  padding: 2rem;
+  padding-bottom: .5rem;
+  padding-right: .5rem;
 }
+
 .profile_popularusers {
-  background: yellow;
   width: 20rem;
```

Comment:
Scratch that...

Use a percentage value for width (remember to `calc` the margin difference.
This will make more sense when all of the `.profile_popularusers` elements are in a single container